libraries/provision/ansible/playbooks/install-sync-gateway-package.yml

```diff
@@ -57,10 +85,14 @@
     couchbase_server_primary_node: "{{ hostvars[groups.couchbase_servers[0]].ansible_host }}"
 
   tasks:
-  - debug: msg="Downloading sync_gateway v. {{ couchbase_sync_gateway_package_url }}"
-  - name: SYNC GATEWAY |  Download sync_gateway rpm
+  - name: SYNC GATEWAY |  Download sync_gateway rpm {{ couchbase_sync_gateway_package_url }}
     get_url: url={{ couchbase_sync_gateway_package_url }} dest=/tmp/{{ couchbase_sync_gateway_package }}
+    when: ansible_distribution == "CentOS"
 
+  - name: SYNC GATEWAY |  Download sync_gateway exe {{ couchbase_sync_gateway_package_url | regex_replace('rpm', 'exe') }}
+    win_get_url: url={{ couchbase_sync_gateway_package_url | regex_replace('rpm', 'exe') }} dest="C:\Users\Administrator\AppData\Local\Temp\{{ couchbase_sync_gateway_package | regex_replace('(.*)rpm$', '\\1exe') }}"
```

Comment:
I think we should redesign how we pass the package name in. This does not seem scalable to other package types